Shared/Protocols/Objectable.swift

```diff
@@ -42,10 +42,8 @@ protocol Objectable: AnyObject {
 
 extension Objectable {
 
-    static func cachedQuerySync(for objectID: String) -> Future<Self, Error> {
-        return Future { promise in
-            promise(.failure(ClientError.generic))
-        }
+    static func cachedQuery(for objectID: String) async throws -> Self {
+        throw ClientError.generic
```

Comment:
Looks like this will always fail for now.

Reply:
Is it being used? If not lets remove.